src/bin/tikv-server.rs

```diff
@@ -403,8 +410,16 @@ fn get_rocksdb_write_cf_option(config: &toml::Value) -> RocksdbOptions {
     opt
 }
 
-fn get_rocksdb_raftlog_cf_option(config: &toml::Value) -> RocksdbOptions {
-    get_rocksdb_cf_option(config, "raftcf", 256 * 1024 * 1024, false, false)
+fn get_rocksdb_raftlog_cf_option(config: &toml::Value, total_mem: u64) -> RocksdbOptions {
+    let mut default_block_cache_size = (total_mem as f64 * 0.01) as u64;
+    if default_block_cache_size < RAFTCF_MIN_MEM {
+        default_block_cache_size = RAFTCF_MIN_MEM;
+    }
+    if default_block_cache_size > RAFTCF_MAX_MEM {
+        default_block_cache_size = RAFTCF_MAX_MEM;
```

Comment:
`default_block_cache_size` takes effect in the condition that `block-cache-size` is not set in config file.